src/main/java/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
```

Comment:
Why is this package out of main package?
```suggestion
package mate.academy.bookstore.exception;
```

Reply:
Because DataProcessingException is a custom exception. I couldn't find the library for it to import